css/loading.css

```
@@ -0,0 +1,90 @@
+@import url(common.css);
+
+body {
+  background-color: rgb(242, 238, 232);
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.text-title {
+  position: absolute;
+  top: 40%;
+  transform: translate(15%, -0%);
+  color: #ffffff;
+
+  z-index: 1;
+}
+
+.row {
+  position: relative;
+  margin: 0 auto;
+}
+
+:before,
+:after {
+  position: absolute;
+  content: '';
+}
+
+.beans {
+  margin: 20px auto;
+  width: 35px;
+  height: 50px;
+  background: #8B4C39;
+  border-radius: 50%;
+  animation: drop2 1s ease-in infinite;
+}
```

저번 HTML, CSS과제에서 멘토님께 받았던 피드백이었는데 이런 식으로 CSS 프로퍼티들을 기능 별로 띄어쓰기 해주면 정말 잘 읽히는 것 같아요!!  한번 해보시면 좋을 것 같습니다 😊 저는 display관련 기능을 올리고 width 속성을 두 번째로 보냈어요 ㅎㅎ 

```suggestion
.beans {
  width: 35px;
  height: 50px;
  
  margin: 20px auto;
 
 border-radius: 50%;
  background: #8B4C39;
  animation: drop2 1s ease-in infinite;
}
```